cli/src/stake.rs

```diff
@@ -684,12 +684,18 @@ impl StakeSubCommands for App<'_, '_> {
                         .takes_value(false)
                         .help("Display inflation rewards"),
                 )
+                .arg(
+                    Arg::with_name("csv")
+                        .long("csv")
+                        .takes_value(false)
+                        .help("Format stake account data in csv")
```

Comment:
```suggestion
                        .help("Format stake rewards data in a CSV table")
```

Reply:
It's slightly confusing how this Arg works in conjunction with `--output`. Perhaps add something to the help that indicates "when `--output` is not set"?
Also, if you find Andrew's point about potential other formats compelling, this arg could be something like:
```
Arg::with_name("rewards_format")
    .long("rewards-fmt")
    .takes_value(true)
    .possible_values(&["csv"])
    .help("Return stake rewards data in specified format when `--output` is not set")
```

Reply:
> Well. It could be another option to --output, like --output=csv instead.

Yeah, I considered that, but `--output` controls the format of the entire response, whereas in this case, we're only formatting the rewards data as CSV. Ultimately, my best suggestion is to keep the 2nd arg, but extend the help text as per above.

Reply:
ok. let's leave it as --csv for now, not that hard to change in the future.

Reply:
I still recommend the change to the help text I suggested above, since it's only the rewards in csv format